src/pages/editor/mind/components/EditorItemPanel/KoniItemPanel.tsx

```diff
@@ -0,0 +1,53 @@
+      />
+      <Item
+        type="node"
+        size="40"
+        shape="koni-custom-node"
+        model={{
+          color: '#B37FEB',
+          label: 'Country',
+          labelOffsetY: 28,
+          icon:
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The key 'icon' is not sorted alphabetically](https://app.codacy.com/manual/afc163/ant-design-pro/pullRequest?prid=4583897)